src/hal-backend-ml-vivante.cc

```diff
@@ -369,6 +369,9 @@ _json_release_neural_network (vivante_handle_s *self)
 {
   if (self->graph) {
     vsi_nn_ReleaseGraph (&self->graph);
+    if (self->graph->ctx) {
+      vsi_nn_ReleaseContext (&self->graph->ctx);
+    }
     self->graph = NULL;
   }
   if (self->ctx) {
```

Comment:
is two time check needed? here?

already you did release above?

Reply:
Right releasing the ctx was redundant. I removed the commit. PTAL